tests/cli/test_work_queues.py

```diff
@@ -579,6 +579,20 @@ def test_ls_with_pool(
             expected_code=0,
         )
 
+    def test_ls_with_zero_concurrency_limit(
+            self,
+            work_queue_1,
+    ):
+        invoke_and_assert(
+            command=f"work-queue set-concurrency-limit {work_queue_1.name} 0",
+            expected_code=0,
+        )
+        invoke_and_assert(
+            command=f"work-queue ls -p {work_queue_1.work_pool.name}",
+            expected_code=0,
+            expected_output_does_not_contain='None',
```

Comment:
could we add a check here that the output _does_ contain 0 instead of `None`?

Reply:
actually - nevermind, because there will be a zero elsewhere which makes the check not worth anything

Reply:
Sure can do!

My little concern is that there might be other places in the `ls` output having 0